### ISSUE TITLE
Build problem with Leia Branch

### ISSUE THREAD

**Issue by technician77** [kodi-pvr/pvr.dvbviewer#79]
First I tried to compile the Matrix branch which succeeded, but then I saw, that 18.X is Leia, so I switched to the Leia branch and used the instructions there. It looked o.k. for the most part but then the error below popped up. I'm not a coder so I can't help myself here.
```
[ 93%] Performing build step for 'pvr.dvbviewer' 
Scanning dependencies of target pvr.dvbviewer 
[ 11%] Building CXX object CMakeFiles/pvr.dvbviewer.dir/src/client.cpp.o 
/home/leonie/pvr.dvbviewer/src/client.cpp:209:11: Fehler: in Konflikt stehende Deklaration für C-Funktion »PVR_ERROR GetEPGForC
hannel(ADDON_HANDLE, int, time_t, time_t)« 
PVR_ERROR GetEPGForChannel(ADDON_HANDLE handle, int iChannelUid, 
          ^~~~~~~~~~~~~~~~ 
In file included from /home/leonie/pvr.dvbviewer/src/client.cpp:29: 
/home/leonie/pvr.dvbviewer/build/build/depends/include/kodi/xbmc_pvr_dll.h:88:13: Anmerkung: vorherige Deklaration von »PVR_ERR
OR GetEPGForChannel(ADDON_HANDLE, const PVR_CHANNEL&, time_t, time_t)« 
  PVR_ERROR GetEPGForChannel(ADDON_HANDLE handle, const PVR_CHANNEL& channel, time_t iStart, time_t iEnd); 
            ^~~~~~~~~~~~~~~~ 
make[5]: *** [CMakeFiles/pvr.dvbviewer.dir/build.make:63: CMakeFiles/pvr.dvbviewer.dir/src/client.cpp.o] Fehler 1 
make[4]: *** [CMakeFiles/Makefile2:73: CMakeFiles/pvr.dvbviewer.dir/all] Fehler 2 
make[3]: *** [Makefile:152: all] Fehler 2 
make[2]: *** [CMakeFiles/pvr.dvbviewer.dir/build.make:114: pvr.dvbviewer-prefix/src/pvr.dvbviewer-stamp/pvr.dvbviewer-build] Fe
hler 2 
make[1]: *** [CMakeFiles/Makefile2:73: CMakeFiles/pvr.dvbviewer.dir/all] Fehler 2 
make: *** [Makefile:84: all] Fehler 2
```

**Comment by manuelm** [comment]
You are mixing build environments. `GetEPGForChannel(ADDON_HANDLE handle, int iChannelUid, ...)` is Matrix signature. `GetEPGForChannel(ADDON_HANDLE handle, const PVR_CHANNEL& channel, ...)` is Leia.

**Comment by technician77** [comment]
Hmm, I think I deleted both the xmbc and the pvr.dvbviewer folder that were created when I cloned the Matrix git. So when I cloned the leia git the were downloaded again. Well, it was very late, maybe there was a copy/paste issue. I'll try again.

**Comment by technician77** [comment]
I used
`git clone  --branch Leia  https://github.com/kodi-pvr/pvr.dvbviewer.git`
instead the command below, which is in the readme for the the Leia branch. That worked. Thanks for the hint. If this was correct, and not some coincidence, you might want to fix the readme.
`git clone https://github.com/kodi-pvr/pvr.dvbviewer.git`

Well, I landed here in the first place because the Arch AUR is not providing a package for LEIA or newer. If you got some time you might be able to fix the situation. A lot of joe-average users like me are not into compiling, also since there is no "auto-update" for self compiled stuff. The current package is here: [https://aur.archlinux.org/packages/kodi-addon-pvr-dvbviewer/](https://aur.archlinux.org/packages/kodi-addon-pvr-dvbviewer/)
